app/views/items/index.html.erb

```diff
@@ -0,0 +1,24 @@
+<%# this is the search bar %>
+<div class="items-index-search">
+  <nav class="navbar navbar-light bg-light">
+    <form class="form-inline">
+      <input class="form-control mr-sm-2" type="search" placeholder="Search" aria-label="Search">
+      <button class="btn btn-outline-success my-2 my-sm-0" type="submit">Search</button>
+    </form>
+  </nav>
+</div>
+<div class="card-and-map">
+  <%# thise are the list of items %>
+  <div class="items-cards-array">
+    <% 15.times do %>
```

Comment:
- Here we should use an each loop to iterate over all the Items, otherwise it will be static. Something like:
`<% @items.each do |item| %>`

Reply:
with the time that I have the placeholder was the best option :)
I will replace this as next push i think tomorrow mornings